etc/sync-function-loader.js

```diff
@@ -1,4 +0,0 @@
-/**
- * DEPRECATED. Please use src/sync-function-loader.js instead.
- */
-module.exports = exports = require('../src/sync-function-loader.js');
```

Comment:
As mentioned in the pull request description, this module was never part of the public API, so there's no need to keep it around.